src/plugins/wizard/opensearch_dashboards.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": "wizard",
+  "version": "1.0.0",
+  "opensearchDashboardsVersion": "opensearchDashboards",
+  "server": true,
+  "ui": true,
+  "requiredPlugins": [
+    "navigation",
+    "charts",
```

Comment:
Why bundles? navigation and charts are plugins.

Reply:
Because we don't actually use/reference their plugin lifecycle methods, we only need to import from them. That is, there's no runtime dependency on them, just a dependency on their bundle. I'm not 100% sure of this distinction, by the way, but it's the intuition I've pieced together from looking at other plugins.

Reply:
Hmm. did not know this. It actually makes sense since i do not need to actually wait for all these plugins to startup to import from them. Lets circle back to this after this PR if thats the case and move the import only plugins to bundles :)